src/list.c

```diff
@@ -313,3 +314,18 @@ int list_longer_than(struct list *list, int count)
 	}
 	return 0;
 }
+
+bool list_search_item(struct list *list, const char *key)
```

Comment:
Either remove the commented-out `string_in_list` function in helpers.c  or uncomment that one and use the existing implementation instead of defining it again here.

Reply:
Oh, I didn't realize there was a function for that already. I'll use the existing one then. Thanks for pointing that out.